Makefile

```
@@ -0,0 +1,20 @@
+.PHONY: all
+all:
+
+.PHONY: tidy
+tidy:
+	black ./
+
+.PHONY: flake8
+flake8:
+	flake8 $$(git ls-files "*.py")
```

Could the git call be replaced with `find . -maxdepth 1 -name "*.py"`? Otherwise the style-check only works if the repository was cloned via git

---

Yeah, we have similar approaches all over the places like https://github.com/os-autoinst/openQA/blob/master/Makefile#L318 so how about

```suggestion
	flake8 $$(git ls-files "*.py" 2>/dev/null ||find . -maxdepth 1 -name "*.py")
```